vendor/vendor.json

```diff
@@ -154,6 +154,12 @@
 			"version": "v12.5.0-beta",
 			"versionExact": "v12.5.0-beta"
 		},
+		{
+			"checksumSHA1": "WMfs+FCE3stapwrAvAS3fjFZlHk=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/operationsmanagement/mgmt/2015-11-01-preview/operationsmanagement",
+			"revision": "21b68149ccf7c16b3f028bb4c7fd0ab458fe308f",
+			"revisionTime": "2018-02-12T16:31:56Z"
```

Comment:
can we update this to include the `version` and `versionExact` fields as above (e.g. by vendoring the tags rather than the commit SHA). This can be done via:

```
govendor update github.com/Azure/azure-sdk-for-go/services/operationsmanagement/mgmt/2015-11-01-preview/operationsmanagement@=v12.5.0-beta
```